aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -2572,17 +2624,39 @@ void dequantize_tensor_per_channel_affine_cpu(
         int64_t channel = rtensor.size(axis);
         auto scales_data = scales.data_ptr<double>();
         auto zero_points_data = zero_points.data_ptr<int64_t>();
+        TORCH_CHECK(
+            qtensor.is_contiguous(qtensor.suggest_memory_format()),
+            "Quantized tensor should be contiguous");
+        TORCH_CHECK(
+            rtensor.is_contiguous(qtensor.suggest_memory_format()),
+            "Float tensor should be contiguous "
+            "in same memory format as quantizd tensor");
```

Comment:
can we create a helper function for these two checks?

Reply:
Just for checks? I know there are a few places we are doing checks, but it seems strange to have a separate two-line function to just do asserts.